content/blog/2016/2016-06-16-parallel-test-executor-plugin.adoc

```diff
@@ -68,6 +68,14 @@ void runTests(def args) {
 void mvn(def args) { /* ... */ }
 ----
 
+[NOTE]
+====
+This pipeline expects to be run as part of a multibranch pipeline.
+To run it stand-alone, replace the `checkout scm` step with
```

Comment:
@bitwiseman my concern is that "to run it standalone" might give people the wrong impression of using "Pipeline from SCM"

Perhaps change the phrasing to:

> This pipeline expects to be run from a `Jenkinsfile`, to copy and paste it directly into Jenkins, replace the `checkout scm` step with: `git 'https://github.com/jenkinsci/git-plugin.git'`